practice-app/libs/create-exercise.js

```diff
@@ -71,92 +71,52 @@ categoryID = 'Q42889';
 
 async function createExercise(wordArray) {
 
+    let nuQue = 4; // Number of questions
+    let nuOpt = 4; // Number of options
```

Comment:
Please remove the comments an put a more understandable variable name instead of comment